ci/gpu/build.sh

```diff
@@ -80,3 +105,8 @@ GTEST_OUTPUT="xml:${WORKSPACE}/test-results/" ./ml_test
 logger "Python py.test for cuML..."
 cd $WORKSPACE/python
 py.test --cache-clear --junitxml=${WORKSPACE}/junit-cuml.xml -v
+
+
+logger "Run ml-prims test..."
+cd $WORKSPACE/ml-prims/build
+GTEST_OUTPUT="xml:${WORKSPACE}/test-results/" ./test/mlcommon_test
```

Comment:
`GTEST_OUTPUT` file path should be different from libcuml tests. From what I know the GTest output filename defaults to `test_detail.xml` and they will override. I would suggest changing libcuml to `"xml:${WORKSPACE}/test-results/libcuml/" and ml-prims to `"xml:${WORKSPACE}/test-results/ml-prims/"`

Reply:
Done :)